src/Car.java

```diff
@@ -0,0 +1,61 @@
+public class Car {
+    private String make = "Tesla";
+    private String model = "Model X";
+    private String color = "Gray";
+    private int doors = 2;
+    private boolean convertible = false;
+
+    public String getMake() {
+        return make;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getDoors() {
+        return doors;
+    }
+
+    public boolean isConvertible() {
+        return convertible;
+    }
+
+    public void setMake(String make) {
+       if (make == null) make = "Unknown";
+       String lowercaseMake = make.toLowerCase();
+        switch (lowercaseMake) {
```

Comment:
You can do `switch(make.toLowerCase()) {`
directly without creating new variable

Reply:
Done